odoo/addons/base/views/res_users_views.xml

```diff
@@ -17,6 +17,27 @@
                 </form>
             </field>
         </record>
+        
+        <record id="revoke_all_devices_form" model="ir.ui.view">
```

Comment:
id should be 'res_users_identitycheck_view_form', we try to standardize naming (model, view, type of view, ...) so that it is easy to find a given view for a given model.